### ISSUE TITLE
Underline_errors NO OPTION error ???

### ISSUE THREAD

**Issue by richlysakowski** [spyder-ide/spyder#18417]
### What steps will reproduce the problem?

I just selected the option "underline_errors" in settings and it gave me this error message for the first time.  

I am using the latest version of Spyder from the WinPython standalone installer. 

My original Anaconda Spyder installation (from 2018) is corrupted beyond repair (and I have been too lazy to remove and reinstall 15 conda enviroments).  

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\ProgramData\WinPython310\python-3.10.4.amd64\lib\site-packages\spyder\plugins\editor\plugin.py", line 419, in _rpc_call
    meth(*args, **kwargs)
  File "C:\ProgramData\WinPython310\python-3.10.4.amd64\lib\site-packages\spyder\plugins\editor\plugin.py", line 1307, in update_source_menu
    state = self.get_option(opt, section=section)
  File "C:\ProgramData\WinPython310\python-3.10.4.amd64\lib\site-packages\spyder\api\plugins\old_api.py", line 111, in get_option
    return super(BasePlugin, self)._get_option(option, default,
  File "C:\ProgramData\WinPython310\python-3.10.4.amd64\lib\site-packages\spyder\plugins\base.py", line 71, in _get_option
    return CONF.get(section, option, default)
  File "C:\ProgramData\WinPython310\python-3.10.4.amd64\lib\site-packages\spyder\config\manager.py", line 460, in get
    raise cp.NoOptionError(option, section)
configparser.NoOptionError: No option ('editor', 'underline_errors') in section: 'editor'
```

## Versions

* Spyder version: 5.3.0 
* Python version: 3.10.4
* Qt version: 5.15.2
* PyQt5 version: 5.15.6
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.31.1;<8.0.0       :  7.32.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.9.0 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.5.0 (OK)
nbconvert >=4.0               :  6.5.0 (OK)
numpydoc >=0.6.0              :  1.2.1 (OK)
paramiko >=2.4.0              :  2.8.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0                :  2.13.7 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.4.1;<1.5.0          :  1.4.1 (OK)
pylsp_black >=1.2.0           :  1.2.1 (OK)
qdarkstyle >=3.0.2;<3.1.0     :  3.0.3 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.1.1 (OK)
qtconsole >=5.3.0;<5.4.0      :  5.3.0 (OK)
qtpy >=2.0.1                  :  2.1.0 (OK)
rtree >=0.9.7                 :  1.0.0 (OK)
setuptools >=49.6.0           :  62.1.0 (OK)
sphinx >=0.6.6                :  4.5.0 (OK)
spyder_kernels >=2.3.0;<2.4.0 :  2.3.0 (OK)
textdistance >=4.2.0          :  4.2.2 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.7 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  0.29.28 (OK)
matplotlib >=3.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.4.2 (OK)
scipy >=0.17.0                :  1.8.0 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @richlysakowski, thanks for reporting. This error was already fixed and will be part of our next version (5.3.2), to be released in a couple of weeks.

For now you can ignore it because it's harmless.

**Comment by richlysakowski** [comment]
Hi Carlos @ccordoba12 , thank you for letting me know.  I noticed that the error has no negative consequences.  Spyder continues to run without any problems after the error happens.
